doc/source/app_abfallplus_de.md

```diff
@@ -141,7 +154,7 @@ The app_id can be found from the url of the play store entry: https://play.googl
 | de.k4systems.leipziglk | Landkreis Leipzig |
 | de.k4systems.abfallappbk | Bad Kissingen |
 | de.cmcitymedia.hokwaste | Hohenlohekreis |
-| de.abfallwecker | Rottweil, Tuttlingen, Waldshut, Prignitz, Nordsachsen |
+| de.abfallwecker | Rottweil, Tuttlingen, Kreis Waldshut, Prignitz, Nordsachsen |
```

Comment:
Will get overwritten by the update_doculinks.py script. Takes information from SUPPORTED_SERVICES in [custom_components/waste_collection_schedule/waste_collection_schedule/service/AppAbfallplusDe.py](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/custom_components/waste_collection_schedule/waste_collection_schedule/service/AppAbfallplusDe.py) which is a generated output from the `generate_supported_services` function (same file).
Need to have a deeper look why it reports this istead of Kreis Waldshut